src/main/java/com/refinedmods/refinedstorage/RSItems.java

```diff
@@ -1,118 +1,153 @@
+        CREATIVE_CONTROLLER = ITEMS.register(RSBlocks.CREATIVE_CONTROLLER.getId().getPath(), () -> new ControllerBlockItem(RSBlocks.CREATIVE_CONTROLLER.get()));
+        createBlockItemFor(RSBlocks.MACHINE_CASING);
+        createBlockItemFor(RSBlocks.CABLE);
+        createBlockItemFor(RSBlocks.DISK_DRIVE);
+        createBlockItemFor(RSBlocks.GRID);
+        createBlockItemFor(RSBlocks.CRAFTING_GRID);
+        createBlockItemFor(RSBlocks.PATTERN_GRID);
+        createBlockItemFor(RSBlocks.FLUID_GRID);
+
+        for (ItemStorageType type : ItemStorageType.values()) {
+            STORAGE_BLOCKS.put(type, ITEMS.register(RSBlocks.STORAGE_BLOCKS.get(type).getId().getPath(),
+                () -> new StorageBlockItem(RSBlocks.STORAGE_BLOCKS.get(type).get())));
+        }
+        for (FluidStorageType type : FluidStorageType.values()) {
+            FLUID_STORAGE_BLOCKS.put(type, ITEMS.register(RSBlocks.FLUID_STORAGE_BLOCKS.get(type).getId().getPath(),
+                () -> new FluidStorageBlockItem(RSBlocks.FLUID_STORAGE_BLOCKS.get(type).get())));
+        }
+
+        createBlockItemFor(RSBlocks.EXTERNAL_STORAGE);
+        createBlockItemFor(RSBlocks.IMPORTER);
+        createBlockItemFor(RSBlocks.EXPORTER);
+        createBlockItemFor(RSBlocks.NETWORK_RECEIVER);
+        createBlockItemFor(RSBlocks.NETWORK_TRANSMITTER);
+        createBlockItemFor(RSBlocks.RELAY);
+        createBlockItemFor(RSBlocks.DETECTOR);
+        createBlockItemFor(RSBlocks.SECURITY_MANAGER);
+        createBlockItemFor(RSBlocks.INTERFACE);
+        createBlockItemFor(RSBlocks.FLUID_INTERFACE);
+        createBlockItemFor(RSBlocks.WIRELESS_TRANSMITTER);
+        createBlockItemFor(RSBlocks.STORAGE_MONITOR);
+        createBlockItemFor(RSBlocks.CONSTRUCTOR);
+        createBlockItemFor(RSBlocks.DESTRUCTOR);
+        createBlockItemFor(RSBlocks.DISK_MANIPULATOR);
+        createBlockItemFor(RSBlocks.CRAFTER);
+        createBlockItemFor(RSBlocks.CRAFTER_MANAGER);
+        createBlockItemFor(RSBlocks.CRAFTING_MONITOR);
+
+        WIRELESS_GRID = ITEMS.register("wireless_grid", () -> new WirelessGridItem(WirelessGridItem.Type.NORMAL));
+        CREATIVE_WIRELESS_GRID = ITEMS.register("creative_wireless_grid", () -> new WirelessGridItem(WirelessGridItem.Type.CREATIVE));
+        WIRELESS_FLUID_GRID = ITEMS.register("wireless_fluid_grid", () -> new WirelessFluidGridItem(WirelessFluidGridItem.Type.NORMAL));
+        CREATIVE_WIRELESS_FLUID_GRID = ITEMS.register("creative_wireless_fluid_grid", () -> new WirelessFluidGridItem(WirelessFluidGridItem.Type.CREATIVE));
+        WIRELESS_CRAFTING_MONITOR = ITEMS.register("wireless_crafting_monitor", () -> new WirelessCraftingMonitorItem(WirelessCraftingMonitorItem.Type.NORMAL));
+        CREATIVE_WIRELESS_CRAFTING_MONITOR = ITEMS.register("creative_wireless_crafting_monitor", () -> new WirelessCraftingMonitorItem(WirelessCraftingMonitorItem.Type.CREATIVE));
+
+
```

Comment:
Maybe it's better to combine all the `createBlockItemFor`s in 1 group

Reply:
I didn't do that here because it would change the order the items appear in the Creative Menu/JEI. 

Keeping is for this PR. Color PR will need to change the order though (or create quite ugly code).